src/components/ReviewCard.jsx

```diff
@@ -1,16 +1,20 @@
+import { Link } from 'react-router-dom'
 export default function ReviewCard ({owner,review_body, category, title, review_img_url, review_id, votes}) {
     return(
         <>
         <br />
+        <Link to={`reviews/${review_id}`}
```

Comment:
You need to add a / infront of reviews in this link, as when I currently click an article on the main page, it takes me to a blank screen and the url in my address bar says /reviews/reviews/:review_id (1 for example).... adding the forward slash will stop that from happening however it needs fixing.